File: decine_java/src/decine/Actor.java
/*  
 *  Fundamentos de Ingenieria del Software
 *  Grado en Ingeniería Informática
 * 
 *  2014 © Copyleft - All Wrongs Reserved
 *
 *  Ernesto Serrano <info@ernesto.es>
 *  
 *  
 * 
 */
package decine;

 class Actor {
    private String nombre;
    private String nacionalidad;
    private String webOficial;
    
    Actor(String nombre, String nacionalidad, String web){
        this.nombre =nombre;
        this.nacionalidad = nacionalidad;
        this.webOficial = web;
    }
    
    protected String getNombre() {     
        return this.nombre;
    }
}

File: decine_cpp2/src/Actor.cpp
#include "Actor.h"

namespace decine {

	Actor::Actor(const std::string &nombre, const std::string &nacionalidad, const std::string &web) {
		this->nombre = nombre;
		this->nacionalidad = nacionalidad;
		this->webOficial = web;
	}

	std::string Actor::getNombre() {
		return this->nombre;
	}
}
